### DISCUSSION TITLE
Some newbie questions

### DISCUSSION THREAD

**Discussion by jarrodCoombes** [keirf/greaseweazle#334] [Q&A]
Hi there, new to the greaseweazle and I am getting it up and running, but I have some questions that I can't seem to get answers for in the Wiki. Thanks in advanced for answering my pesky questions.

1. What does "unknown mark f<something>" mean? I seem to only get that when reading 1.2Mb disks on a 1.2Mb drive. When I do 360Kb disks on a 360kb drive I don't seem to ever see this error. The images I create still seem to work fine, even when this comes up, and formatting the disk on a DOS machine works fine too (no bad sectors).

2. How do you read that dot grid that comes up at the end? The dots are successful reads and the x means the sector could not be read?

3. The floppy power header on the board is to power a 3.5" floppy drive? Or is it to power the actual greasweazle?

4. I have not yet been able to write an image, when I try I get an error message `Failed to verify Track 0.0` but in DOS I can format and write to the disk, so I am pretty sure it's not a bad disk or drive.

**Comment by keirf** [comment]
1. It is a sign that the track is not reading well. Possibly the drive density setting is incorrect. 5.25-inch drives often have to be told the density, via pin 2. So try setting pin 2 to low voltage as follows: `gw pin set 2 L`
2. Yes dot is good, x is bad
3. To power a 3.5-inch drive. Other drives inc 5.25 inch drives need external psu
4. Could be the same problem as (1)
